### ISSUE TITLE
Up and Down hotkeys for Rides and Paths

### ISSUE THREAD

**Issue by PFCKrutonium** [OpenRCT2/OpenRCT2#2412]
Basically a hot key to change to up and down, and on rides, pressing it twice will select a steeper option if available.

**Comment by e-foley** [comment]
In general, it would be pretty nifty to be able to build a tracked ride without having to click anywhere.

**Comment by jackinloadup** [comment]
Agree, I always wish there was a hotkey to select the directional arrows and such in the path tool. Ride building as well would be pretty awesome.

**Comment by PFCKrutonium** [comment]
Should I change the title of the issue to "Hotkey all the things"? :P

**Comment by IntelOrca** [comment]
@PFCKrutonium no

**Comment by PFCKrutonium** [comment]
I figured as much ;)
